charts/opentelemetry-operator/examples/default/rendered/deployment.yaml

```diff
@@ -22,10 +22,10 @@ spec:
     metadata:
       annotations:
         kubectl.kubernetes.io/default-container: manager
-        
+        aaa: bbb
```

Comment:
here `nindent` render spaces if `podAnnotations` is empty

Reply:
you can fix it in  https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-operator/templates/deployment.yaml#L19